packages/cli/generators/import-lb3-models/index.js

```diff
@@ -136,6 +148,20 @@ Learn more at https://loopback.io/doc/en/lb4/Importing-LB3-models.html
     );
     debug('LB4 model data', templateData);
 
+    if (!templateData.isModelBaseBuiltin) {
+      const baseName = templateData.modelBaseClass;
+      if (
+        !this.existingLb4ModelNames.includes(baseName) &&
```

Comment:
Just want to make sure that I understand it correctly, the reason we check the existence of the base model is because we need to import the custom base class and also it's parent class if that doesn't exist?

Reply:
Yes.

Let's say the application has:
- `AuditedPersistedModel` which is adding auditing functionality for all CRUD operations provided by `PersistedModel`,
- several models like `Product`, `Category` and `Store`, where all models are based on `AuditedPersistedModel`.

When we run import for the first time and choose only `Store` to import, we also need to import `AuditedPersistedModel` to allow the new LB4 model to inherit from the correct base class (model).

When we run import for the second time and choose to import `Product` and `Category`, we should not import `AuditedPersistedModel` again, because that could overwrite bits imported manually in the previous step. Instead, LB4 CLI will skip importing this base model.